From: You-Sheng Yang <vicamo@gmail.com>
Date: Thu, 29 May 2025 15:20:15 +0800
Subject: wl: add MODULE_DESCRIPTION

MODULE_DESCRIPTION is now a mandatory meta info for a module since
v6.15-rc1 commit 6c6c1fc09de35 ("modpost: require a
MODULE_DESCRIPTION()").

Bug-Ubuntu: https://bugs.launchpad.net/bugs/2111898
Signed-off-by: You-Sheng Yang <vicamo@gmail.com>
---
 amd64/src/wl/sys/wl_linux.c | 1 +
 i386/src/wl/sys/wl_linux.c  | 1 +
 2 files changed, 2 insertions(+)

diff --git a/amd64/src/wl/sys/wl_linux.c b/amd64/src/wl/sys/wl_linux.c
index cbf4efe..d36f8bc 100644
--- a/amd64/src/wl/sys/wl_linux.c
+++ b/amd64/src/wl/sys/wl_linux.c
@@ -167,6 +167,7 @@ static void wl_report_radio_state(wl_info_t *wl);
 #endif
 
 MODULE_LICENSE("MIXED/Proprietary");
+MODULE_DESCRIPTION("Broadcom STA Wireless driver");
 
 static struct pci_device_id wl_id_table[] =
 {
diff --git a/i386/src/wl/sys/wl_linux.c b/i386/src/wl/sys/wl_linux.c
index 0397c3c..b519dff 100644
--- a/i386/src/wl/sys/wl_linux.c
+++ b/i386/src/wl/sys/wl_linux.c
@@ -162,6 +162,7 @@ static void wl_report_radio_state(wl_info_t *wl);
 #endif
 
 MODULE_LICENSE("MIXED/Proprietary");
+MODULE_DESCRIPTION("Broadcom STA Wireless driver");
 
 static struct pci_device_id wl_id_table[] =
 {
